src/ParaTest/Runners/PHPUnit/ResultPrinter.php

```diff
@@ -321,15 +338,14 @@ protected function getDefects(array $defects, $type)
     /**
      * Prints progress for large test collections
      */
-    protected function printProgress()
+    protected function getProgress()
     {
-        printf(
-            ' %' . $this->numTestsWidth . 'd (%3s%%)',
+        return sprintf(
+            ' %' . $this->numTestsWidth . 'd / %' . $this->numTestsWidth . 'd (%3s%%)',
             $this->casesProcessed,
-            floor(($this->casesProcessed / $this->totalCases) * 100)
+            $this->totalCases,
```

Comment:
Interesting option. Is it save to say that every tests that doesn't show up in the junit-logs are skipped or incomplete? That way we could even print S for incompletes or skipped tests and fix #60 ...